crawl-ref/source/initfile.cc

```diff
@@ -4982,6 +4982,8 @@ void game_options::write_webtiles_options(const string& name)
 
     tiles.json_write_bool("show_game_time", Options.show_game_time);
 
+    tiles.json_write_bool("action_panel_disabled",
+            Options.action_panel.empty());
```

Comment:
This is fine for now but in the long run it may be worth adding a real option that positively disables the action panel. I think I'd rather do it by converting `action_panel_show` into a yes/no/never thing, but I think that will need some further work on the options API.